### PR TITLE
Update index.html (minor, polish)

### PR DESCRIPTION
**Pull request by TallTed**
punctuation, typos, line breaks...

### CODE REVIEW

**Review by brentzundel — APPROVED** [review]

**Comment by TallTed** [comment]
resolved conflicts; hopefully this can merge soon, and then I can resolve conflicts with my later similar effort...

File: index.html [hunk 1]
```diff
@@ -416,15 +430,30 @@ <h3>Timeline</h3>
...
-        <p>Additional technical coordination with the following Groups will be made, per the <a href="https://www.w3.org/Consortium/Process/#WGCharter">W3C Process Document</a>:</p>
+        <p>
+Additional technical coordination with the following Groups will be made, per the 
+<a href="https://www.w3.org/Consortium/Process/#WGCharter">W3C Process Document</a>:
+        </p>
 
         <div>
           <h3 id="w3c-coordination">W3C Groups</h3>
           <dl>
             <dt><a href="https://www.w3.org/2017/vc/WG/">Verifiable Claims Working Group</a></dt>
```
Comment (msporny) [inline index.html]: Looks like this snuck back in there after we updated... May need this rebased.

**Comment by msporny** [comment]
Merge conflicts, all of this is editorial, Ted, please fix the merge conflicts and we need to get this in ASAP so all other PRs can be rebased on top of it.

**Comment by TallTed** [comment]
@msporny - I think that's got it.

**Review by msporny — APPROVED** [review]
Editorial. Merging.

**Pull request merged**